### PR TITLE
refactor(react-query-devtools): use `superjson` to display values

### LINKED ISSUE #1552: Error displays as an empty object
When my query function throws an error, the devtools displays it as an empty object.
I would expect it to display the `message` and maybe `stack` string properties that the object contains.

If I throw a custom error with additional properties, such as `status`, it shows up just fine in the devtools. So it seems to only be the default error properties that are not being shown.

image

### PR DESCRIPTION
**Pull request by aboqasem**
Fixes #1552

### CODE REVIEW

File: package.json
Comment (TkDodo) [inline package.json:115]: This needs to be a `dependency` of the `react-query-devtools` package please.

**Review by TkDodo — CHANGES_REQUESTED** [review]

**Comment by TkDodo** [comment]
not sure that there should be so many changes in package-lock.json, also, builds are failing. I would suggest reverting the changes to packge-lock (restoring what we have in main), then running `npm install` again.

**Comment by aboqasem** [comment]
I tried and it produces similar result. Which version of `npm` should I use? Currently I have to `npm i --legacy-peer-deps` on `8.18.0`.

**Comment by TkDodo** [comment]
our `.nvmrc` uses node `v16.14.2`, so please use that

**Review by TkDodo — APPROVED** [review]

**Comment by TkDodo** [comment]
sorry, I wanted to wait until we merged our beta branch with bundling changes. If you fix the merge conflicts, I'll merge this PR

**Review by TkDodo — APPROVED** [review]

**Comment by TkDodo** [comment]
yeah, we fixed it immediately. sorry 🙈

**Pull request merged**
